.circleci/config.yml

```diff
@@ -62,6 +63,23 @@ commands:
               --justfile ../../../nested.just \
               simulate council
 
+  notify-failures-on-develop:
+    description: "Notify Slack"
+    parameters:
+      channel:
+        type: string
+        default: C03N11M0BBN
```

Comment:
What channel is this? Presumably #notify-ci-failures?

Reply:
yes exactly https://oplabs-pbc.slack.com/archives/C03N11M0BBN